app/src/test/java/com/jalasoft/routesapp/ui/auth/registerUser/viewModel/RegisterUserViewModelTest.kt

```diff
@@ -21,18 +21,18 @@ internal class RegisterUserViewModelTest : TestCase() {
     @BeforeEach
     public override fun setUp() {
         super.setUp()
-        viewModel = RegisterUserViewModel()
+        // viewModel = RegisterUserViewModel()
     }
 
     @Test
     fun testForValidateFieldsSuccess() {
-        val result = viewModel.validateFields(names, email, password, confirmPassword)
-        assertEquals("", result)
+        // val result = viewModel.validateFields(names, email, password, confirmPassword)
```

Comment:
remove commented code if its not being used

Reply:
Fixed on Rodrigo's pull Request